pkg/volume/plugins.go

```diff
@@ -146,6 +151,8 @@ const (
 	// Name of a volume in external cloud that is being provisioned and thus
 	// should be ignored by rest of Kubernetes.
 	ProvisionedVolumeName = "placeholder-for-provisioning"
+	// Mount options annotations
+	MountOptionAnnotation = "volume.beta.kubernetes.io/mountOptions"
```

Comment:
We specced it to be `mountOptions` in design document.  I am open to changing the annotation. Is there a convention for this?